src/main/java/io/pinecone/configs/PineconeConnection.java

```diff
@@ -116,17 +135,18 @@ private static Metadata assembleMetadata(PineconeClientConfig clientConfig) {
         return metadata;
     }
 
-    static String getEndpoint(PineconeClientConfig clientConfig, PineconeConnectionConfig connectionConfig) {
-        String endpoint = (connectionConfig.getConnectionUrl() != null) ?
-                connectionConfig.getConnectionUrl().replaceFirst("https?://", "") :
-                String.format("%s-%s.svc.%s.pinecone.io",
-                        connectionConfig.getIndexName(),
-                        clientConfig.getProjectName(),
-                        clientConfig.getEnvironment());
-
-        logger.debug("Pinecone endpoint is: " + endpoint);
+    public static String getEndpoint(String host) {
```

Comment:
Updated the getEndpoint() by removing the old method of constructing the gRPC endpoint using indexname, projectName, and environment.

Reply:
Sure!